pom.xml

```diff
@@ -93,6 +120,9 @@
 							<failsOnError>true</failsOnError>
 							<includeTestSourceDirectory>true</includeTestSourceDirectory>
 							<propertyExpansion>basedir=${project.basedir}</propertyExpansion>
+							<excludes>**/DateFormatStrategy.java,**/DateFormatStrategy.java,
```

Comment:
Is it simple to fix these formatting changes? I see that these files are included without modification, are they excluded since we do not want to modify them?

Reply:
It's because the headers don't match our header, happy to discuss an alternative or we can change our header?

Reply:
This is fine for any classes that we have wholesale. Please add a HTML comment though to explain why these are excluded:

`<!-- Excluded as these are external dropins -->`